src/Module/Admin/Federation.php

```diff
@@ -14,168 +14,94 @@ public static function content(array $parameters = [])
 	{
 		parent::content($parameters);
 
-		// get counts on active friendica, diaspora, redmatrix, hubzilla, gnu
-		// social and statusnet nodes this node is knowing
-		//
-		// We are looking for the following platforms in the DB, "Red" should find
-		// all variants of that platform ID string as the q() function is stripping
-		// off one % two of them are needed in the query
-		// Add more platforms if you like, when one returns 0 known nodes it is not
-		// displayed on the stats page.
-		$platforms = ['Friendi%%a', 'Diaspora', '%%red%%', 'Hubzilla', 'BlaBlaNet', 'GNU Social', 'StatusNet', 'Mastodon', 'Pleroma', 'socialhome', 'ganggo'];
-		$colors = [
-			'Friendi%%a' => '#ffc018', // orange from the logo
-			'Diaspora'   => '#a1a1a1', // logo is black and white, makes a gray
-			'%%red%%'    => '#c50001', // fire red from the logo
-			'Hubzilla'   => '#43488a', // blue from the logo
-			'BlaBlaNet'  => '#3B5998', // blue from the navbar at blablanet-dot-com
-			'GNU Social' => '#a22430', // dark red from the logo
-			'StatusNet'  => '#789240', // the green from the logo (red and blue have already others
-			'Mastodon'   => '#1a9df9', // blue from the Mastodon logo
-			'Pleroma'    => '#E46F0F', // Orange from the text that is used on Pleroma instances
-			'socialhome' => '#52056b', // lilac from the Django Image used at the Socialhome homepage
-			'ganggo'     => '#69d7e2', // from the favicon
+		// get counts on active federation systems this node is knowing
+		// We list the more common systems by name. The rest is counted as "other"
+		$systems = [
+			'Friendica'   => ['name' => 'Friendica', 'color' => '#ffc018'], // orange from the logo
+			'diaspora'    => ['name' => 'Diaspora', 'color' => '#a1a1a1'], // logo is black and white, makes a gray
+			'red'         => ['name' => 'Red Matrix', 'color' => '#c50001'], // fire red from the logo
+			'hubzilla'    => ['name' => 'Hubzilla', 'color' => '#43488a'], // blue from the logo
+			'gnusocial'   => ['name' => 'GNU Social', 'color' => '#a22430'], // dark red from the logo
+			'statusnet'   => ['name' => 'StatusNet', 'color' => '#789240'], // the green from the logo (red and blue have already others
+			'mastodon'    => ['name' => 'Mastodon', 'color' => '#1a9df9'], // blue from the Mastodon logo
+			'pleroma'     => ['name' => 'Pleroma', 'color' => '#E46F0F'], // Orange from the text that is used on Pleroma instances
+			'socialhome'  => ['name' => 'SocialHome', 'color' => '#52056b'], // lilac from the Django Image used at the Socialhome homepage
+			'wordpress'   => ['name' => 'WordPress', 'color' => '#016087'], // Background color of the homepage
```

Comment:
Is wordpress only counted when speaking AP or also as `feed` contacts?

Reply:
These should be systems with a working federation, so most likely they will speak AP.